modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/service/impl/PowwowMeetingOccurrenceLocalServiceImpl.java

```diff
@@ -73,6 +73,10 @@ public PowwowMeetingOccurrence addPowwowMeetingOccurrence(
 		return powwowMeetingOccurrence;
 	}
 
+	public int countByPowwowMeetingId(long powwowMeetingId) {
```

Comment:
let's not rely on this method. In the view, we'll use method `List.size()`, because:
- the number of occurrences is small enough, maximum 50.
- there will be another logic that we may need to reduce the size of available list. we'll discuss further on this.